docker/automations/bots/bath-lights.js

```diff
@@ -32,6 +38,136 @@ module.exports = (name, {
         let toggledTimer = null
         let unlockedTimer = null
 
+        // Command verification configuration (safe defaults = disabled)
+        const config = {
+            verification: commandConfig.verification || 0,     // 0 = disabled
+            maxRetries: commandConfig.maxRetries || 0,         // 0 = disabled
+            retryDelay: commandConfig.retryDelay || 1000,      // 1 second between retries
+        }
+
+        // Pending commands tracking (only used if verification enabled)
+        const pendingCommands = new Map() // reason -> {expectedState, attempts, timer, etc}
+
+        // Smart publish: legacy mode (direct) or verification mode
+        const smartPublish = (topic, payload, reason = 'unknown') => {
+            if (config.verification === 0 || config.maxRetries === 0) {
+                // Legacy mode - direct publish (existing behavior)
+                mqtt.publish(topic, payload)
+                return
+            }
+
+            // Verification mode - use state-based verification
+            const expectedState = payload.state
+            
+            // Cancel any existing command for the same reason to avoid conflicts
```

Comment:
as the command reason is always different, this allows to have two pending commands each with different target state. Rather there should be only one pending command for lights state change